src/internal/server/PrintLeftoverLabelHandler_test.go

```diff
@@ -0,0 +1,134 @@
+package server_test
+
+import (
+	"bytes"
+	"io"
+	"main/internal/server"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+type testParameters_PrintLeftoverLabelHandler struct {
+	reqMethod          string
+	reqBody            io.Reader
+	expectedStatusCode int
+	expectedMessage    string
+}
+
+func TestPrintLeftoverLabelHandler(t *testing.T) {
+	// define test cases
+	var testParams = []testParameters_PrintLeftoverLabelHandler{
```

Comment:
I love these tests!

Did you use some resource as an example here?

Reply:
@jlaari I didn't actually - I came up with that on my own!